projects/crud/src/lib/components/open-tec-popup/open-tec-popup.component.html

```diff
@@ -0,0 +1,7 @@
+<div class="container" id="confirmation-dialog">
+  <div mat-dialog-content>Would you like to capture an Identity Card or a Passport?</div>
+  <div class="actions">
+    <button *ngFor="let option of options" mat-button id="scan-passport-btn" (click)="dialogAction(option)">{{ option |
+      translate }}</button>
+  </div>
+</div>
```

Comment:
@lamiselzobeidy the message and the buttons need to be generic and passed to the dialog